packages/target-nodejs-sdk/src/index.js

```diff
@@ -223,6 +235,10 @@ export default function bootstrap(fetchApi) {
 
       const visitor = createVisitor(options, this.config);
 
+      const request = this.telemetryProvider.executeTelemetries(
```

Comment:
Nitpick:  I think for sake of clarity we should assign the resulting request back to options.request, just so it's more obvious that executeTelemetries is simply modifying the incoming options.request.  And then when assigning to targetOptions below we can just have `...options`

Reply:
i'll disable the eslint error, it would be reassigning a parameter's property